src/main/java/Decide.java

```diff
@@ -143,20 +143,20 @@ public void decide() {
     }
   };
 
-  public Main() {
+  public Decide() {
     this.parameters = new Parameters();
     decide();
   }
 
   public static void main(String args[]) {
-    Main main = new Main();
+    Decide decide = new Decide();
   }
 
   private COMPTYPE doubleCompare(double a, double b) {
     if (Math.abs(a - b) < 0.000001)
       return COMPTYPE.EQ;
     if (a < b)
       return COMPTYPE.LT;
-    return COMPTYPE.GT;
+    return COMPTYPE.GQ;
```

Comment:
```suggestion
    return COMPTYPE.GT;
```
Should be GT, right?

Reply:
Nice catch, I'm committing a fix that changes the enum on both places! 👍